python/ray/util/timer.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import time
+
+
+class _Timer:
```

Comment:
This class is just moved from rllib/utils to be accessible to ray/util.

Reply:
:+1: